src/_pages/ProjectDetails/Overview/components/OverviewPanel/index.tsx

```diff
@@ -34,13 +36,23 @@ export default function OverviewPanel({ project }: Props) {
   }
 
   return (
-    <Card background="base" className="flex h-fit flex-col divide-y divide-greyscale-50/8 p-0 lg:p-0">
-      <ProjectOverviewLead projectId={project?.id} projectLeads={projectLeads} projectInvited={projectInvited} />
-      <ProjectOverviewContributor contributorCount={contributorCount} topContributors={topContributors} />
-      <ProjectOverviewEcosystem ecosystems={ecosystems} />
-      <ProjectOverviewSponsors sponsors={sponsors} />
-      <ProjectOverviewTechnologies technologies={technologies} />
-      <ProjectOverviewMoreInfo moreInfos={moreInfos} />
-    </Card>
+    <>
+      <Card background="base" className="flex h-fit flex-col divide-y divide-greyscale-50/8 p-0 lg:p-0">
```

Comment:
Not sure, would have to test.

```suggestion
      <Card background="base" className="flex h-fit flex-col divide-y divide-greyscale-50/8" hasPadding={false}>
```